content/700-optimize/400-recommendations/300-full-table-scans-caused-by-like-operations.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Full table scans caused by LIKE operations'
+metaTitle: 'Optimize Recommendations: Full table scans caused by LIKE operations'
+metaDescription: "Learn about the recommendation provided by Optimize for full table scans caused by Like operations."
+tocDepth: 3
+toc: true
+---
+
+The following query targeting the `User` model provides `contains` and `endsWith` as options, which translate to `LIKE` and `ILIKE` SQL operators.
+
+```jsx
+await prisma.user.findMany({ 
+  where: { 
+    email: { contains: "gmail.com" }, 
+    name: { endsWith: "Burk" } 
+  } 
+})
+```
+
+## Why is this a problem?
+
+`LIKE` and `ILIKE` operators in SQL can lead to full table scans, potentially impacting performance, especially with larger datasets:
+
+### UX
+
+- **Slower load times:** Full table scans can significantly increase the time it takes to retrieve data, leading to longer wait times for users.
+
+### Resource utilization
+
+- **Increased resource usage:** Full table scans increase CPU, memory usage, and disk I/O, straining system resources for your database.
+- **Increased costs:** In serverless database pricing plans, more intensive resource usage can translate into higher costs.
+
+## Recommendation
```

Comment:
yea remove this

Reply:
@mhessdev could you make a *suggestion* with what to remove; The bullet points or the recommendation? I'm slightly confused on this.